notification-service/src/test/java/com/redhat/parodos/notification/dto/NotificationAssemblerDTOTest.java

```diff
@@ -0,0 +1,60 @@
+package com.redhat.parodos.notification.dto;
+
+import com.redhat.parodos.notification.jpa.entity.NotificationMessage;
+import com.redhat.parodos.notification.jpa.entity.NotificationRecord;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+
+import java.time.Instant;
+import java.util.Collections;
+import java.util.List;
+import java.util.UUID;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class NotificationAssemblerDTOTest {
+
+	@Test
+	void toModel() {
+		NotificationRecord entity = Mockito.mock(NotificationRecord.class);
+		NotificationMessage notificationMessage = Mockito.mock(NotificationMessage.class);
+
+		String messageBody = "message-body-test";
+		Instant messageCreation = Instant.now();
```

Comment:
To improve test quality, best practices recommend to do not use `Instant.now()` because you can't assert the exact value since it's different in each execution.
I understand that in this situation is not a critical point, but I'd like to suggest anyway to use something like `Instant.parse("2023-03-01T10:30:00.00Z")` to have a clean codebase.

Reply:
I use this in the beginning of the test so I can use the same Instant for all steps
In this case it doesn't matter what is the value assigned, the validation is done on the object itself, the same like when I generate random UUID
I can change it, but I don't think it must